crm_lead_lost_reason/wizard/lost_reason.py

```diff
@@ -28,10 +28,17 @@ class CrmLeadLost(models.TransientModel):
     _name = 'crm.lead.lost'
     _description = __doc__
 
+    def _default_reason(self):
+        active_id = self._context.get('active_id', [])
+        if active_id:
+            lead = self.env['crm.lead'].browse(active_id)
```

Comment:
@gurneyalex thanks, ok it's safer to check the active_model.
in the context, the user marks lost leads one by one.